mim/utils/utils.py

```diff
@@ -327,12 +336,37 @@ def get_installed_path(package: str) -> str:
         return osp.join(pkg.location, package2module(package))
 
 
-def get_torch_cuda_version() -> Tuple[str, str]:
-    """Get PyTorch version and CUDA version if it is available.
+def is_npu_available() -> bool:
+    """Returns True if Ascend PyTorch and npu devices exist."""
+    return IS_NPU_AVAILABLE
+
+
+def get_npu_version() -> str:
+    """Returns the version of NPU when npu devices exist."""
+    if not is_npu_available():
+        return ''
+    ascend_home_path = os.environ.get('ASCEND_HOME_PATH', None)
+    if not ascend_home_path or not os.path.exists(ascend_home_path):
+        raise RuntimeError(
+            highlighted_error(
+                f'ASCEND_HOME_PATH:{ascend_home_path} does not exists when '
+                'installing OpenMMLab projects on Ascend NPU.'
+                "Please run 'source set_env.sh' in the CANN installation path."
+            ))
+    npu_version = torch_npu.get_cann_version(ascend_home_path)
+    return npu_version
+
+
+def get_torch_device_version() -> Tuple[str, str, str]:
+    """Get PyTorch version and CUDA/NPU version if it is available.
 
     Example:
-        >>> get_torch_cuda_version()
-        '1.8.0', '102'
+        >>> get_torch_device_version()
+        '1.8.0', 'cpu', ''
+        >>> get_torch_device_version()
+        '1.8.0', 'cuda', '102'
+        >>> get_torch_device_version()
+        '1.11.0', 'ascend', '6.0.2'
```

Comment:
Can we modify `6.0.2` to `602` which is consistent with cuda 102.

Reply:
Thanks for the suggestion, it has been modified